forge-gui/res/cardsfolder/upcoming/kardum,_patron_of_flames.txt

```diff
@@ -0,0 +1,19 @@
+Name:Kardum, Patron of Flames
+ManaCost:2 R R
+Types:Legendary Creature Demon
+PT:4/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, exile the top card of your library face down. (You can't look at it.)
+SVar:TrigCounter:DB$ PutCounter | CounterType$ FLAME | CounterNum$ 1 | ValidCards$ Self | SubAbility$ DBSeek
+SVar:DBSeek:DB$ ChangeZone | Origin$ Library | Destination$ Exile | ExileFaceDown$ True | AtRandom$ True | NoShuffle$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ 1 | ChangeType$ Card.cmcEQX+YouCtrl | RememberChanged$ True 
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigReturn | TriggerDescription$ When CARDNAME dies, put the cards exiled with it into their owner's hand.
+SVar:TrigReturn:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered+ExiledWithSource | RememberChanged$ True | Origin$ Exile | Destination$ Hand | SubAbility$ DBDelay
+SVar:DBDelay:DB$ DelayedTrigger | DelayedTriggerDefinedPlayer$ You | Mode$ Phase | Phase$ End of Turn | Execute$ TrigDiscardExiled | RememberObjects$ Remembered | TriggerDescription$ At the beginning of the end step of your next turn, discard those cards.
+SVar:TrigDiscardExiled:DB$ Discard | Mode$ Defined | DelayedTriggerDefinedPlayer$ You | DefinedCards$ DelayTriggerRemembered | TriggerDescription$ Discard the cards put into your hand with CARDNAME
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Exile | Static$ True | ValidCard$ Card.Self | Execute$ DBForget
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Library | Static$ True | ValidCard$ Card.Self | Execute$ DBForget
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Hand | Static$ True | ValidCard$ Card.Self | Execute$ DBForget
+SVar:DBForget:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Count$CardCounters.FLAME
+AI:RemoveDeck:Random
```

Comment:
I don't think this is needed - card fuels itself just fine

This tag is generally for cards that require fuel from elsewhere (a creature that sucks unless your deck has lots of Gates, for example) or that are very sideboardy-specific hate cards (a card that does nothing unless your opponent controls a Swamp)

Reply:
It should have the HasAttackEffect line though